address-controller/src/main/resources/swagger.json

```diff
@@ -0,0 +1,640 @@
+                    "$ref": "#/definitions/ObjectMeta"
+                },
+                "spec": {
+                    "$ref": "#/definitions/AddressSpec"
+                },
+                "status": {
+                    "$ref": "#/definitions/AddressStatus"
+                }
+            }
+        },
+        "AddressMeta": {
+            "type": "object",
+            "properties": {
+                "name": {
+                    "type": "string"
+                },
+                "addressSpace": {
+                    "type": "string"
+                },
+                "uuid": {
+                    "type": "string"
+                }
+            }
+        },
+        "AddressSpec": {
+            "type": "object",
+            "required": [
+                "type"
+            ],
+            "properties": {
+                "type": {
+                    "$ref": "#/definitions/AddressType"
+                },
+                "plan": {
+                    "type": "string"
+                },
+                "address": {
+                    "type": "string"
+                }
+            }
+        },
+        "AddressStatus": {
+            "type": "object",
+            "properties": {
+                "isReady": {
+                    "type": "boolean"
+                },
+                "messages": {
+                    "type": "array",
+                    "items": {
+                        "type": "string"
+                    }
+                }
+            }
+        },
+        "AddressList": {
+            "type": "object",
+            "required": [
+                "apiVersion",
+                "kind",
+                "items"
+            ],
+            "properties": {
+                "apiVersion": {
+                    "type": "string",
+                    "default": "enmasse.io/v1",
+                    "enum": [
+                        "enmasse.io/v1"
+                    ]
+                },
+                "kind": {
+                    "type": "string",
+                    "enum": [
+                        "AddressList"
+                    ]
+                },
+                "items": {
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/Address"
+                    }
+                }
+            }
+        },
+        "ObjectMeta": {
+            "type": "object",
+            "required": [
+                "name"
+            ],
+            "properties": {
+                "name": {
+                    "type": "string"
+                },
+                "namespace": {
+                    "type": "string"
+                }
+            }
+        },
+        "AddressType": {
+            "type": "string",
+            "description": "Type of address (queue, topic, ...)",
+            "enum": [
+                "QUEUE",
+                "TOPIC",
+                "ANYCAST",
+                "MULTICAST"
```

Comment:
I guess this is already an EnMasseism.  But can you remind me what the difference is between a Topic and Mulitcast address.  In Artemis a Topic is based on a multicast address.

Reply:
Multicast is what we've named the 'direct' multicast supported by the qpid dispatch router.